golang/cli/vin.go

```diff
@@ -1,17 +1,31 @@
 package main
```

Comment:
Create `vin_test.go` and move in related tests.

Reply:
these are now officially untested "example" code :)